pages/bindingDescription.xml

```diff
@@ -71,6 +71,142 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <p>You can always further specify this information within one
         of the <gi>decoNote</gi> elements within
         <gi>binding</gi>.</p>
-        </body>
+
+    <div type="level3">
+      <head>Boards</head>
+        <p>In the
+            <tag>decoNote type="Boards"</tag>
+            you can specify using keywords the wood type (if known) and
+            encode the presence of a textile inlay or of a mirror.
+        </p>
+    </div>
+
+    <div type="level3">
+      <head>Cover</head>
+       <p>In the
+             <tag>decoNote type="Cover"</tag>, using the keywords,
+             you can describe the the blind-tooling decoration
+             and record the presence of an additional leather patch.
+         </p>
+     </div>
+
+     <div type="level3">
+       <head>Sewing Stations</head>
+         <p>The
+             <tag>decoNote type="SewingStations"</tag>
+             requires to type a number, which has to be equal to the number
+             of sewing stations (not to the number of pairs of sewing stations).
+         </p>
+
+         <p>A further description of the sewing is possible
+             within a general <gi>decoNote</gi> with a progressive
+             <att>xml:id</att>. Here, if identified, you can specify with keywords the sewing pattern,
+             following Bozzacchi's classification <ref target="https://www.zotero.org/groups/358366/ethiostudies/items/itemKey/2968T6TC/q/bozzacchi"></ref>
+             and the origin of sewing thread.
```

Comment:
following Bozzacchi and the origin 

or

the sawing pattern and the origin of the sawing thread ???

I am not sure, if the second is right I would anticipate perhaps